### ISSUE TITLE
BibLaTeX Hover and Completion ignore Globs in `\addbibresource{}`

### ISSUE THREAD

**Issue by Janfel** [James-Yu/LaTeX-Workshop#3937]
## Pre-checks
- [x] The issue has not been reported in this repository.
- [x] The issue remains after disabling all other extensions and restarting Visual Studio Code.
- [x] The [FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) cannot address the issue.
- [x] The issue is not related to compiling a document, or the document can be successfully compiled in the OS terminal but not in Visual Studio Code with this extension.

## Environment
- Operating System: Manjaro Linux 22.1.3 Talos
- Visual Studio Code Version: 1.78.2
- LaTeX Workshop Version: 9.11.5
- TeX Distribution Version: TeX Live 2023 (Manjaro `extra` Repository)

## The Issue
I am using BibLaTeX with `biber`, which allows me to use globs in `\addbibresource{}` as below.

```latex
\addbibresource[glob]{bibliography/*.bib}
```

This works fine during compilation, but the files in `bibliography/` are completely ignored when it comes to code completion and hovering.

### Reproduction Steps
1. Set up `test.tex` to use `biblatex` and `biber`.
2. Create a bibliography file `bibliography/test.bib` containing a citation `Test2023`.
3. Reference the bibliography file in `test.tex` using `\addbibresource[glob]{bibliography/*.bib}`.
4. Try to `\cite{Test2023}`.
5. Code completion and hovering over `Test2023` will not work.

### Expected Behavior
- Code completion for `Test2023` inside `\cite{}` works as normal.
- Hovering over `Test2023` inside `\cite{Test2023}` works as normal.

### LaTeX Workshop Output
```
[03:48:22.560][Logger] New log placeholder %WS1% registered for /home/janfel/REDACTED/Seminararbeit .
[03:48:22.560][Config] latex-workshop.latex.outDir: "latex_build" .
[03:48:22.578][Config] latex-workshop.latex.autoBuild.run: "never" .
[03:48:22.596][Config] latex-workshop.synctex.afterBuild.enabled: true .
[03:48:22.596][Config] latex-workshop.linting.chktex.enabled: true .
[03:48:22.597][Config] latex-workshop.linting.lacheck.enabled: true .
[03:48:22.633][Config] latex-workshop.bibtex-format.trailingComma: true .
[03:48:22.636][Manager] Set $LATEXWORKSHOP_DOCKER_LATEX: ""
[03:48:22.647][Server] Creating LaTeX Workshop http and websocket server.
[03:48:22.687][Server] Server successfully started: {"address":"127.0.0.1","family":"IPv4","port":44147} .
[03:48:22.697][Extension] Initializing LaTeX Workshop.
[03:48:22.698][Extension] Extension root: /home/janfel/.local/share/code-oss/extensions/james-yu.latex-workshop-9.11.5
[03:48:22.698][Extension] $PATH: /usr/local/sbin:/usr/local/bin:/usr/bin:/opt/cuda/bin:/opt/cuda/nsight_compute:/opt/cuda/nsight_systems/bin:/var/lib/flatpak/exports/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl
[03:48:22.698][Extension] $SHELL: /bin/zsh
[03:48:22.699][Extension] $LANG: de_DE.UTF-8
[03:48:22.699][Extension] $LC_ALL: undefined
[03:48:22.699][Extension] process.platform: linux
[03:48:22.699][Extension] process.arch: x64
[03:48:22.699][Extension] vscode.env.appName: Visual Studio Code
[03:48:22.699][Extension] vscode.env.remoteName: undefined
[03:48:22.700][Extension] vscode.env.uiKind: 1
[03:48:22.700][Extension] LaTeX Workshop initialized.
[03:48:22.706][Format][Bib] Bibtex format config: {"tab":"  ","case":"lowercase","left":"{","right":"}","trailingComma":true,"sort":["key"],"alignOnEqual":true,"sortFields":false,"fieldsOrder":[],"firstEntries":["string","xdata"]}
[03:48:22.707][Extension] Trigger characters for intellisense of LaTeX documents: ["\\",",","{"]
[03:48:22.709][Manager] Current workspace folders: ["file://%WS1%"]
[03:48:22.710][Manager] Current workspaceRootDir: file://%WS1%
[03:48:22.811][Server] valdOrigin is http://127.0.0.1:44147
[03:48:22.844][Viewer][Panel] Restoring at column 1 with state {"pdfFileUri":"file://%WS1%/latex_build/Seminararbeit.pdf","scale":"auto","scrollMode":0,"sidebarView":0,"spreadMode":0,"scrollTop":999,"scrollLeft":0,"trim":0,"synctexEnabled":true,"autoReloadEnabled":true}.
[03:48:22.893][Viewer][Panel] Internal PDF viewer at http://127.0.0.1:44147/viewer.html?file=pdf..ZmlsZSUzQSUyRiUyRiUyRmhvbWUlMkZqYW5mZWwlMkZEb2t1bWVudGUlMkZTdHVkaXVtJTJGRmVybnVuaSUyMEhhZ2VuJTJGS3Vyc2UlMkYyMDIzJTIwU1MlMkYxOTI0JTIwQmFjaGVsb3JzZW1pbmFyJTNBJTIwTW9kZWxsaWVydW5nJTIwdW5kJTIwVmVyaWZpa2F0aW9uJTJGQXVzYXJiZWl0dW5nJTJGU2VtaW5hcmFyYmVpdCUyRmxhdGV4X2J1aWxkJTJGU2VtaW5hcmFyYmVpdC5wZGY .
[03:48:22.901][Cacher][Watcher] Watched %WS1%/latex_build/Seminararbeit.pdf with a new watcher on %WS1%/latex_build .
[03:48:22.901][Event] FILE_WATCHED: "%WS1%/latex_build/Seminararbeit.pdf"
[03:48:23.135][Cacher] Cache not finished on %WS1%/Seminararbeit.tex when getting its children.
[03:48:23.140][Cacher] Caching %WS1%/Seminararbeit.tex .
[03:48:23.141][Cacher] Updated inputs of %WS1%/Seminararbeit.tex .
[03:48:23.141][Cacher] Parse LaTeX AST with fast-parse: %WS1%/Seminararbeit.tex .
[03:48:23.236][Cacher] Parsed LaTeX AST with LU in 94.42 ms: %WS1%/Seminararbeit.tex .
[03:48:23.247][Cacher] Parsed LaTeX AST in 10.98 ms: %WS1%/Seminararbeit.tex .
[03:48:23.297][DupLabel] Checking for duplicate labels: %WS1%/Seminararbeit.tex .
[03:48:23.297][Cacher] Updated elements of %WS1%/Seminararbeit.tex .
[03:48:23.298][Cacher] Bib %WS1%/test.bib from %WS1%/Seminararbeit.tex .
[03:48:23.298][Intelli][Citation] Parsing .bib entries from %WS1%/test.bib
[03:48:23.299][Intelli][Citation] Parse BibTeX AST from %WS1%/test.bib .
[03:48:23.299][Cacher][Watcher] Watched %WS1%/test.bib with a new watcher on %WS1% .
[03:48:23.299][Event] FILE_WATCHED: "%WS1%/test.bib"
[03:48:23.299][Cacher][Path] Cannot resolve bibliography/*.bib .
[03:48:23.300][Cacher] Updated bibs of %WS1%/Seminararbeit.tex .
[03:48:23.300][Cacher] Cached %WS1%/Seminararbeit.tex .
[03:48:23.301][Event] FILE_PARSED: "%WS1%/Seminararbeit.tex"
[03:48:23.301][Structure] Structure cleared on undefined root.
[03:48:23.308][Event] STRUCTURE_UPDATED
[03:48:23.308][Manager] Found files that might be root, choose the first one: %WS1%/Seminararbeit.tex .
[03:48:23.309][Manager] Root file changed: from undefined to %WS1%/Seminararbeit.tex
[03:48:23.309][Manager] Start to find all dependencies.
[03:48:23.309][Manager] Root file languageId: latex
[03:48:23.309][Event] ROOT_FILE_CHANGED: "%WS1%/Seminararbeit.tex"
[03:48:23.309][Cacher][Watcher] Reset.
[03:48:23.313][Cacher] Adding %WS1%/Seminararbeit.tex .
[03:48:23.314][Cacher][Watcher] Watched %WS1%/Seminararbeit.tex with a new watcher on %WS1% .
[03:48:23.314][Event] FILE_WATCHED: "%WS1%/Seminararbeit.tex"
[03:48:23.315][Cacher] Caching %WS1%/Seminararbeit.tex .
[03:48:23.315][Cacher] Updated inputs of %WS1%/Seminararbeit.tex .
[03:48:23.316][Cacher] Parse LaTeX AST with fast-parse: %WS1%/Seminararbeit.tex .
[03:48:23.316][Event] ROOT_FILE_SEARCHED
[03:48:23.321][Linter] ChkTeX lints root %WS1%/Seminararbeit.tex .
[03:48:23.322][Linter][ChkTeX] Linter for ChkTeX command The command is chktex:["-wall","-n22","-n30","-e16","-q","-f%f:%l:%c:%d:%k:%n:%m\n","%WS1%/Seminararbeit.tex"].
[03:48:23.329][Linter] LaCheck lints root %WS1%/Seminararbeit.tex .
[03:48:23.330][Linter][LaCheck] Linter for LaCheck command The command is lacheck:["%WS1%/Seminararbeit.tex"].
[03:48:23.352][Linter] Linter for root successfully finished in 0s 7ms
[03:48:23.353][Linter][LaCheck] Logged 0 messages.
[03:48:23.366][Intelli][Citation] Parsed 1 bib entries from %WS1%/test.bib .
[03:48:23.367][Event] FILE_PARSED: "%WS1%/test.bib"
[03:48:23.368][Linter] Linter for root failed with exit code 2 and error:
[03:48:23.368][Linter][ChkTeX] No .chktexrc file is found to determine TabSize.
[03:48:23.375][Linter][ChkTeX] Logged 1 messages.
[03:48:23.552][Cacher] Parsed LaTeX AST with LU in 235.90 ms: %WS1%/Seminararbeit.tex .
[03:48:23.562][Cacher] Parsed LaTeX AST in 10.10 ms: %WS1%/Seminararbeit.tex .
[03:48:23.638][DupLabel] Checking for duplicate labels: %WS1%/Seminararbeit.tex .
[03:48:23.639][Cacher] Updated elements of %WS1%/Seminararbeit.tex .
[03:48:23.640][Cacher] Bib %WS1%/test.bib from %WS1%/Seminararbeit.tex .
[03:48:23.640][Cacher][Path] Cannot resolve bibliography/*.bib .
[03:48:23.641][Cacher] Updated bibs of %WS1%/Seminararbeit.tex .
[03:48:23.641][Cacher] Cached %WS1%/Seminararbeit.tex .
[03:48:23.641][Event] FILE_PARSED: "%WS1%/Seminararbeit.tex"
[03:48:23.642][Cacher] Parsing .fls %WS1%/latex_build/Seminararbeit.fls .
[03:48:23.665][Cacher] Found .aux %WS1%/Seminararbeit.tex from .fls %WS1%/latex_build/Seminararbeit.fls , parsing.
[03:48:23.666][Cacher] Parsed .aux %WS1%/Seminararbeit.tex .
[03:48:23.666][Cacher] Parsed .fls %WS1%/latex_build/Seminararbeit.fls .
[03:48:23.682][Structure] Structure force updated with 1 root sections for %WS1%/Seminararbeit.tex .
[03:48:23.683][Event] STRUCTURE_UPDATED
[03:48:23.683][Structure] Structure force updated with 1 root sections for %WS1%/Seminararbeit.tex .
[03:48:23.683][Event] STRUCTURE_UPDATED
[03:48:23.818][Viewer] Handle data type: open
[03:48:24.105][Server] Preview PDF file: file://%WS1%/latex_build/Seminararbeit.pdf
[03:48:24.241][Viewer] Handle data type: loaded
[03:48:24.241][Event] VIEWER_PAGE_LOADED
[03:48:24.241][Viewer] SyncTex after build invoked.
[03:48:24.242][Locator] No active editor found.
[03:48:24.450][Event] VIEWER_STATUS_CHANGED: {"pdfFileUri":"file://%WS1%/latex_build/Seminararbeit.pdf","scale":"auto","scrollMode":0,"sidebarView":0,"spreadMode":0,"scrollTop":999,"scrollLeft":0,"trim":0,"synctexEnabled":true,"autoReloadEnabled":true}
[03:48:24.450][Event] VIEWER_STATUS_CHANGED: {"pdfFileUri":"file://%WS1%/latex_build/Seminararbeit.pdf","scale":"auto","scrollMode":0,"sidebarView":0,"spreadMode":0,"scrollTop":999,"scrollLeft":0,"trim":0,"synctexEnabled":true,"autoReloadEnabled":true}
[03:48:24.452][Event] VIEWER_STATUS_CHANGED: {"pdfFileUri":"file://%WS1%/latex_build/Seminararbeit.pdf","scale":"auto","scrollMode":0,"sidebarView":0,"spreadMode":0,"scrollTop":999,"scrollLeft":0,"trim":0,"synctexEnabled":true,"autoReloadEnabled":true}
[03:48:24.458][Event] VIEWER_STATUS_CHANGED: {"pdfFileUri":"file://%WS1%/latex_build/Seminararbeit.pdf","scale":"auto","scrollMode":0,"sidebarView":0,"spreadMode":0,"scrollTop":999,"scrollLeft":0,"trim":0,"synctexEnabled":true,"autoReloadEnabled":true}
[03:48:24.458][Event] VIEWER_STATUS_CHANGED: {"pdfFileUri":"file://%WS1%/latex_build/Seminararbeit.pdf","scale":"auto","scrollMode":0,"sidebarView":0,"spreadMode":0,"scrollTop":999,"scrollLeft":0,"trim":0,"synctexEnabled":true,"autoReloadEnabled":true}
[03:48:24.460][Event] VIEWER_STATUS_CHANGED: {"pdfFileUri":"file://%WS1%/latex_build/Seminararbeit.pdf","scale":"auto","scrollMode":0,"sidebarView":0,"spreadMode":0,"scrollTop":999,"scrollLeft":0,"trim":0,"synctexEnabled":true,"autoReloadEnabled":true}
[03:48:25.200][Manager] Current workspace folders: ["file://%WS1%"]
[03:48:25.215][Manager] Found root file from active editor: %WS1%/Seminararbeit.tex
[03:48:25.216][Manager] Keep using the same root file: %WS1%/Seminararbeit.tex
[03:48:25.216][Structure] Structure updated with 1 root sections for %WS1%/Seminararbeit.tex .
[03:48:25.216][Event] ROOT_FILE_SEARCHED
[03:48:25.217][Event] STRUCTURE_UPDATED
[03:48:25.217][Linter] ChkTeX lints root %WS1%/Seminararbeit.tex .
[03:48:25.218][Linter][ChkTeX] Linter for ChkTeX command The command is chktex:["-wall","-n22","-n30","-e16","-q","-f%f:%l:%c:%d:%k:%n:%m\n","%WS1%/Seminararbeit.tex"].
[03:48:25.225][Linter] LaCheck lints root %WS1%/Seminararbeit.tex .
[03:48:25.226][Linter][LaCheck] Linter for LaCheck command The command is lacheck:["%WS1%/Seminararbeit.tex"].
[03:48:25.238][Linter] Linter for root successfully finished in 0s 6ms
[03:48:25.239][Linter][LaCheck] Logged 0 messages.
[03:48:25.257][Linter] Linter for root failed with exit code 2 and error:
[03:48:25.258][Linter][ChkTeX] No .chktexrc file is found to determine TabSize.
[03:48:25.258][Linter][ChkTeX] Logged 1 messages.
[03:48:52.301][Event] VIEWER_STATUS_CHANGED: {"pdfFileUri":"file://%WS1%/latex_build/Seminararbeit.pdf","scale":"auto","scrollMode":0,"sidebarView":0,"spreadMode":0,"scrollTop":980,"scrollLeft":0,"trim":0,"synctexEnabled":true,"autoReloadEnabled":true}
[03:48:52.307][Event] VIEWER_STATUS_CHANGED: {"pdfFileUri":"file://%WS1%/latex_build/Seminararbeit.pdf","scale":"auto","scrollMode":0,"sidebarView":0,"spreadMode":0,"scrollTop":998,"scrollLeft":0,"trim":0,"synctexEnabled":true,"autoReloadEnabled":true}
```

### Developer Tools Console
```
 WARN Via 'product.json#extensionEnabledApiProposals' extension 'ms-vscode.vscode-selfhost-test-provider' wants API proposal 'testContinuousRun' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
log.ts:431  WARN Settings pattern "security.*" doesn't match any settings
log.ts:431  WARN SettingsEditor2: Settings not included in settingsLayout.ts: interactiveEditor.editMode, accessibility.verbosity.diff-editor, accessibility.verbosity.terminal
log.ts:431  WARN Settings pattern "security.*" doesn't match any settings
TMScopeRegistry.ts:45 Overwriting grammar scope name to file mapping for scope text.tex.
Old grammar file: file:///opt/visual-studio-code/resources/app/extensions/latex/syntaxes/TeX.tmLanguage.json.
New grammar file: file:///home/janfel/.local/share/code-oss/extensions/james-yu.latex-workshop-9.11.5/syntax/TeX.tmLanguage.json
register @ TMScopeRegistry.ts:45
TMScopeRegistry.ts:45 Overwriting grammar scope name to file mapping for scope text.tex.latex.
Old grammar file: file:///opt/visual-studio-code/resources/app/extensions/latex/syntaxes/LaTeX.tmLanguage.json.
New grammar file: file:///home/janfel/.local/share/code-oss/extensions/james-yu.latex-workshop-9.11.5/syntax/LaTeX.tmLanguage.json
register @ TMScopeRegistry.ts:45
TMScopeRegistry.ts:45 Overwriting grammar scope name to file mapping for scope text.bibtex.
Old grammar file: file:///opt/visual-studio-code/resources/app/extensions/latex/syntaxes/Bibtex.tmLanguage.json.
New grammar file: file:///home/janfel/.local/share/code-oss/extensions/james-yu.latex-workshop-9.11.5/syntax/Bibtex.tmLanguage.json
register @ TMScopeRegistry.ts:45
TMScopeRegistry.ts:45 Overwriting grammar scope name to file mapping for scope text.tex.markdown_latex_combined.
Old grammar file: file:///opt/visual-studio-code/resources/app/extensions/latex/syntaxes/markdown-latex-combined.tmLanguage.json.
New grammar file: file:///home/janfel/.local/share/code-oss/extensions/james-yu.latex-workshop-9.11.5/syntax/markdown-latex-combined.tmLanguage.json
register @ TMScopeRegistry.ts:45
TMScopeRegistry.ts:45 Overwriting grammar scope name to file mapping for scope source.cpp.embedded.latex.
Old grammar file: file:///opt/visual-studio-code/resources/app/extensions/latex/syntaxes/cpp-grammar-bailout.tmLanguage.json.
New grammar file: file:///home/janfel/.local/share/code-oss/extensions/james-yu.latex-workshop-9.11.5/syntax/cpp-grammar-bailout.tmLanguage.json
register @ TMScopeRegistry.ts:45
log.ts:421  INFO [perf] Render performance baseline is 36ms
2log.ts:431  WARN Settings pattern "security.*" doesn't match any settings
log.ts:431  WARN Settings pattern "security.*" doesn't match any settings
DevTools failed to load source map: Could not load content for http://127.0.0.1:44147/out/viewer/latexworkshop.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:44147/out/viewer/components/connection.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:44147/out/viewer/components/htmleditor.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:44147/out/viewer/components/synctex.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:44147/out/viewer/components/utils.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:44147/out/viewer/components/pagetrimmer.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:44147/out/viewer/components/externalpromise.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:44147/out/viewer/components/viewerhistory.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:44147/build/pdf.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:44147/viewer/viewer.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:44147/build/pdf.worker.js.map: Fetch through target failed: Target not supported; Fallback: Unknown error
```

## Anything Else?
- The globbing feature was introduced in BibLaTeX 3.13 (2019-08-17).

**Comment by jlelong** [comment]
Can you check the content of `latex_build/Seminararbeit.fls` and look for anything mentioning your `.bib` files?
Although we do not support globing inside `\addbibresource`, all the dependencies should be correctly resolved after building thanks to the content of the `.fls` generated by the LaTeX compiler itself.

**Comment by Janfel** [comment]
Nothing in `latex_build/Seminararbeit.fls`, but they are listed in `latex_build/Seminararbeit.blg`. I’ve attached the file in the Details block below.

<details><pre>
[0] Config.pm:307> INFO - This is Biber 2.19
[0] Config.pm:310> INFO - Logfile is 'latex_build/Seminararbeit.blg'
[75] biber:340> INFO - === Fr Jun 23, 2023, 10:43:06
[89] Biber.pm:419> INFO - Reading 'latex_build/Seminararbeit.bcf'
[147] Biber.pm:979> INFO - Found 2 citekeys in bib section 0
[158] Biber.pm:4419> INFO - Processing section 0
[158] Utils.pm:85> INFO - Globbing data source 'bibliography/*.bib'
[158] Utils.pm:96> INFO - Globbed data source 'bibliography/*.bib' to 'bibliography/10.1007_978-3-319-10575-8_7-citation.bib,bibliography/IEEE Xplore Citation BibTeX Download 2023.6.18.4.47.20.bib'
[168] Biber.pm:4610> INFO - Looking for bibtex file 'bibliography/10.1007_978-3-319-10575-8_7-citation.bib' for section 0
[169] bibtex.pm:1713> INFO - LaTeX decoding ...
[172] bibtex.pm:1519> INFO - Found BibTeX data source 'bibliography/10.1007_978-3-319-10575-8_7-citation.bib'
[216] Biber.pm:4610> INFO - Looking for bibtex file 'bibliography/IEEE Xplore Citation BibTeX Download 2023.6.18.4.47.20.bib' for section 0
[217] bibtex.pm:1713> INFO - LaTeX decoding ...
[219] bibtex.pm:1519> INFO - Found BibTeX data source 'bibliography/IEEE Xplore Citation BibTeX Download 2023.6.18.4.47.20.bib'
[231] UCollate.pm:68> INFO - Overriding locale 'de-DE' defaults 'variable = shifted' with 'variable = non-ignorable'
[231] UCollate.pm:68> INFO - Overriding locale 'de-DE' defaults 'normalization = NFD' with 'normalization = prenormalized'
[231] Biber.pm:4239> INFO - Sorting list 'nyt/global//global/global' of type 'entry' with template 'nyt' and locale 'de-DE'
[231] Biber.pm:4245> INFO - No sort tailoring available for locale 'de-DE'
[237] bbl.pm:660> INFO - Writing 'latex_build/Seminararbeit.bbl' with encoding 'UTF-8'
[240] bbl.pm:763> INFO - Output to latex_build/Seminararbeit.bbl
</pre></details>

**Comment by James-Yu** [comment]
The issue is resolved in 718180e . You may try out the package or wait for the next release.
